kubecost-exporter.md

```diff
@@ -0,0 +1,117 @@
+# Running Kubecost as a Prometheus metric exporter
+
+Running Kubecost as a Prometheus metric exporter allows you to export various cost metrics to Prometheus without setting up any other Kubecost dependencies. Doing so lets you write PromQL queries to calculate the cost and efficiency of any Kubernetes concept, e.g. namespace, service, label, deployment, etc. You can also calculate the cost of different Kubernetes resources, e.g. nodes, PVs, LoadBalancers, and more. Finally, you can do other interesting things like create custom alerts via AlertManager and custom dashboards via Grafana. 
+
+## Installing
+
+> Note: all deployments of Kubecost function as a Prometheus metric exporter. We strongly recommend helm as an install path to take advantage of Kubecost’s full potential. [View recommended install](http://docs.kubecost.com/install).
+
+If you would prefer to not use the recommended install option and just deploy the Kubecost open source cost model as a metric exporter, you can follow these steps:
+
+
+1. Apply the combined YAML:
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/kubecost/cost-model/develop/kubernetes/exporter/exporter.yaml --namespace cost-model
+    ```
+
+    > If you want to use a namespace other than `cost-model`, you will have to edit the `ClusterRoleBinding` after applying the YAML to change `subjects[0].namespace`. You can do this with `kubectl edit clusterrolebinding cost-model`.
+
+2. To verify that metrics are available:
+
+    ```
+    kubectl port-forward --namespace cost-model service/cost-model 9003
+    ```
+
+    Visit [http://localhost:9003/metrics](http://localhost:9003/metrics) to see exported metrics
+
+Add Kubecost scrape config to Prom ([more](https://prometheus.io/docs/introduction/first_steps/#configuring-prometheus))
+```
+- job_name: cost-model
+  scrape_interval: 1m
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+    - targets: [‘cost-model.cost-model.:9003/metrics’]
+```
+
+Done! Kubecost is now exporting cost metrics. See the following sections for different metrics available and query examples.
+
+## Available Prometheus Metrics 
+
+| Metric       | Description                                                                                            |
+| ------------ | ------------------------------------------------------------------------------------------------------ |
+| node_cpu_hourly_cost | Hourly cost per vCPU on this node  |
+| node_gpu_hourly_cost | Hourly cost per GPU on this node  |
+| node_ram_hourly_cost   | Hourly cost per Gb of memory on this node                       |
```

Comment:
We now have `kubecost_load_balancer_cost` to add to this list, yeah?

Reply:
I'm not seeing it with the latest (public) `cost-model` container image. Is that a feature coming in this release?

Reply:
Should be available for the last 5+ releases: https://github.com/kubecost/cost-model/blob/f845eb02c200c4da993827ad2d62bf47b5922f84/pkg/costmodel/metrics.go#L636

Reply:
Ah, it's probably not there for me because I don't have any load balancers so its not being emitted.

Reply:
Are we documenting these metrics somewhere other than PROMETHEUS.md? It also doesn't have `kubecost_load_balancer_cost`, not to mention others like:
* `kubecost_node_is_spot`
* `kubecost_network_zone_egress_cost`
* `kubecost_network_region_egress_cost`
* `kubecost_network_internet_egress_cost`
* `kubecost_cluster_management_cost`
* `kubecost_cluster_info`

Reply:
We have not documented anywhere yet! In this section, it seems like `kubecost_load_balancer_cost` and `kubecost_cluster_management_cost` are the most relevant? 

I haven't added network_egress metrics yet primarily because they are in a different repo.

Reply:
I'll open a follow-up PR to add these metrics both here and to PROMETHEUS.md